.examples/delay-startup/Makefile

```diff
@@ -0,0 +1,43 @@
+BINARY=gatus
+
+# Because there's a folder called "test", we need to make the target "test" phony
+.PHONY: test
+
+install:
+	go build -mod vendor -o $(BINARY) .
+
+run:
+	GATUS_CONFIG_PATH=./config.yaml ./$(BINARY)
+
+delay_and_run:
+	GATUS_CONFIG_PATH=./config.yaml GATUS_DELAY_START_SECONDS=5 ./$(BINARY)
+
+clean:
+	rm $(BINARY)
+
+test:
+	go test ./... -cover
+
+
+##########
+# Docker #
+##########
+
+docker-build:
+	docker build -t twinproduction/gatus:latest .
+
+docker-run:
+	docker run -p 8080:8080 --name gatus twinproduction/gatus:latest
+
+docker-build-and-run: docker-build docker-run
+
+
+#############
+# Front end #
+#############
+
+frontend-build:
+	npm --prefix web/app run build
+
+frontend-run:
+	npm --prefix web/app run serve
```

Comment:
By example, I meant more like a Dockerfile similar to https://github.com/TwiN/gatus/tree/master/.examples/docker-minimal but with the environment variable you just added.

That being said, now that I've given it more thoughts, perhaps it would be easier to just add a section in the README under FAQ with a title like `Configuring a startup delay` that mentions the new `GATUS_DELAY_START_SECONDS` environment variable.

Reply:
Done in 636b570 and 7b224ca